hack/make.ps1

```diff
@@ -303,7 +303,10 @@ Function Run-UnitTests() {
 # Start of main code.
 Try {
     Write-Host -ForegroundColor Cyan "INFO: make.ps1 starting at $(Get-Date)"
-    $root=$(pwd)
+
+    # Get to the root of the repo
+    $root = $(Split-Path $MyInvocation.MyCommand.Definition -Parent | Split-Path -Parent)
+    Push-Location $root
```

Comment:
Never mind; looks like it actually changes to the specified location; https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.management/push-location